src/databricks/labs/remorph/snow/databricks_preview.py

```diff
@@ -0,0 +1,24 @@
+import logging
+from typing import ClassVar
+
+from sqlglot import expressions as exp
+
+from databricks.labs.remorph.snow.databricks import Databricks
+
+logger = logging.getLogger(__name__)
+
+
+class DatabricksPreivew(Databricks):
+    databricks = Databricks()
```

Comment:
why it's not injected through constructor?

Reply:
this is default setup inside sqlglot